### ISSUE TITLE
isGroup is undefined & group chat being identified as PrivateChat

### ISSUE THREAD

**Issue by notedwin-dev** [pedroslopez/whatsapp-web.js#3390]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this a problem caused by your code, or is it specifically because of the library?

- [X] I have double-checked my code carefully.

### Describe the bug.

isGroup is undefined & group chat being identified as PrivateChat.

Workaround:
```js
const { Client, LocalAuth } = require("whatsapp-web.js");

const client = new Client({
    puppeteer: {
      headless: true,
      args: ["--no-sandbox", "--disable-setuid-sandbox"],
    },
    authStrategy: new LocalAuth(),
    webVersionCache: {
      type: "remote",
      remotePath:
        "https://raw.githubusercontent.com/wppconnect-team/wa-version/main/html/2.3000.1018501131-alpha.html",
    },
  });
  
client.on("message", async (msg) => {
    const { from } = msg;
    const chat = await msg.getChat();
    let contact = await chat.getContact();

    if (contact.isGroup === true) {
      console.log("Ignoring group message");
      return;
    } else {
      console.log("Message received:", msg.body);
     }
 });

### Expected Behavior

I expected chat.isGroup to be true or false, instead, I got undefined.
```
Chat: PrivateChat {
  id: {
    server: 'g.us',
    user: 'aaa',
    _serialized: 'aaa@g.us'
  },
  name: '',
  isGroup: undefined,
  isReadOnly: false,
  unreadCount: 0,
  timestamp: 1732716353,
  archived: undefined,
  pinned: false,
  isMuted: undefined,
  muteExpiration: 0,
  lastMessage: Message {
    _data: {
      id: [Object],
      viewed: false,
      body: 'hi',
      type: 'chat',
      t: 1732716421,
      notifyName: 'aaa',
      from: [Object],
      to: [Object],
      author: [Object],
      ack: 1,
      invis: false,
      isNewMsg: true,
      star: false,
      kicNotified: false,
      recvFresh: true,
      isFromTemplate: false,
      pollInvalidated: false,
      isSentCagPollCreation: false,
      latestEditMsgKey: null,
      latestEditSenderTimestampMs: null,
      mentionedJidList: [],
      groupMentions: [],
      isEventCanceled: false,
      eventInvalidated: false,
      isVcardOverMmsDocument: false,
      isForwarded: false,
      hasReaction: false,
      viewMode: 'VISIBLE',
      messageSecret: [Object],
      productHeaderImageRejected: false,
      lastPlaybackProgress: 0,
      isDynamicReplyButtonsMsg: false,
      isCarouselCard: false,
      parentMsgId: null,
      isMdHistoryMsg: false,
      stickerSentTs: 0,
      isAvatar: false,
      lastUpdateFromServerTs: 0,
      invokedBotWid: null,
      bizBotType: null,
      botResponseTargetId: null,
      botPluginType: null,
      botPluginReferenceIndex: null,
      botPluginSearchProvider: null,
      botPluginSearchUrl: null,
      botPluginSearchQuery: null,
      botPluginMaybeParent: false,
      botReelPluginThumbnailCdnUrl: null,
      botMsgBodyType: null,
      requiresDirectConnection: false,
      bizContentPlaceholderType: null,
      hostedBizEncStateMismatch: false,
      senderOrRecipientAccountTypeHosted: false,
      placeholderCreatedWhenAccountIsHosted: false,
      links: []
    },
    mediaKey: undefined,
    id: {
      fromMe: false,
      remote: 'aaa@g.us',
      id: '3A179859486398FC30D6',
      participant: [Object],
      _serialized: 'false_120363371704224503@g.us_3A179859486398FC30D6_60175503585@c.us'
    },
    ack: 1,
    hasMedia: false,
    body: 'hi',
    type: 'chat',
    timestamp: 1732716421,
    from: 'aaa@g.us',
    to: 'aaa@c.us',
    author: 'aaa@c.us',
    deviceType: 'ios',
    isForwarded: false,
    forwardingScore: 0,
    isStatus: false,
    isStarred: false,
    broadcast: undefined,
    fromMe: false,
    hasQuotedMsg: false,
    hasReaction: false,
    duration: undefined,
    location: undefined,
    vCards: [],
    inviteV4: undefined,
    mentionedIds: [],
    groupMentions: [],
    orderId: undefined,
    token: undefined,
    isGif: false,
    isEphemeral: undefined,
    links: []
  }
}
Message received: hi
```

### Steps to Reproduce the Bug or Issue

```js
const { Client, LocalAuth } = require("whatsapp-web.js");

const client = new Client({
    puppeteer: {
      headless: true,
      args: ["--no-sandbox", "--disable-setuid-sandbox"],
    },
    authStrategy: new LocalAuth(),
    webVersionCache: {
      type: "remote",
      remotePath:
        "https://raw.githubusercontent.com/wppconnect-team/wa-version/main/html/2.3000.1018501131-alpha.html",
    },
  });
  
client.on("message", async (msg) => {
    const { from } = msg;
    const chat = await msg.getChat();
    console.log("Chat:", chat);

    if (chat.isGroup === true) {
      console.log("Ignoring group message");
      return;
    } else {
      console.log("Message received:", msg.body);
    }
   });
   ```

### WhatsApp Account Type

Standard

### Browser Type

Google Chrome (Mac OS)

### Operation System Type

Mac OS

### Phone OS Type

Android

### WhatsApp-Web.js Version

1.26.0

### WhatsApp Web Version

2.3000.1018501131

### Node.js Version

v21.5.0

### Authentication Strategy

LocalAuth

**Comment by Halphas** [comment]
same here setsubject etc. therefore stopped working. god....

**Comment by alechkos** [comment]
Fixed in #3380 

Use
```bash
npm install whatsapp-web.js@v1.26.1-alpha.3
```
